src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -368,6 +369,82 @@ public boolean isInGoal(int x, int y, int z) {
             logDirect("tunneling");
             return true;
         }
+        if(msg.startsWith("tunnelcustom"))
+        {
+            String suffix = msg.substring("tunnelcustom".length());
+            int a;
+            int b;
+            int c;
+            if (suffix.isEmpty()) {
+                    logDirect("Need to specify depth, height and width of the tunnel. eg .b tunnel2 depth height width");
+                    return true;
+            }
+            else {
+                try {
+                    String[] spl = suffix.split(" ");
+                    if(spl.length != 4) {
+                        logDirect(Integer.toString(spl.length));
+                        logDirect("Please enter 3 values");
+                        return true;
+                    }
+                    a = Integer.parseInt(spl[1]);
+                    b = Integer.parseInt(spl[2]);
+                    c = Integer.parseInt(spl[3]);
+                } catch (NumberFormatException | ArrayIndexOutOfBoundsException | NullPointerException ex) {
+                    logDirect("unable to parse");
+                    return true;
+                }
+            }
+            if(c < 1)
+            {
+                logDirect("Width must at least be 1 block");
+                return true;
+            }
+            if(b < 2)
+            {
+                logDirect("Height must at least be 2 blocks");
+                return true;
+            }
+
+            //Adjustment for player location
+            a --;
+            b --;
+            c --;
+            BlockPos corner1 = ctx.playerFeet();
+            BlockPos corner2 = ctx.playerFeet();
+            EnumFacing enumfacing = ctx.player().getHorizontalFacing();
+            int addition = 1;
+            //Check if width is uneven to center player's location
+            if(c % 2 ==0)
```

Comment:
Formatting Specification: Boolean operators should contain spaces between its operands.